### PR TITLE
fixed rpath handling for Mac OS X

### PR DESCRIPTION
**Pull request by malter**

### CODE REVIEW

File: lib/orogen/templates/typekit/CMakeLists.txt [hunk 1]
```diff
@@ -105,8 +105,8 @@ set_target_properties(${libname} PROPERTIES LINK_INTERFACE_LIBRARIES ${OrocosRTT
 set_target_properties(${libname} PROPERTIES INTERFACE_LINK_LIBRARIES ${OrocosRTT_LIBRARIES})
 if(WITH_RPATH AND APPLE)
   set_target_properties( ${libname} PROPERTIES
-    INSTALL_NAME_DIR "@rpath"
-    LINK_FLAGS "-Wl,-rpath ${CMAKE_INSTALL_PREFIX}/lib -rpath ${CMAKE_INSTALL_PREFIX}/lib/orocos -rpath ${CMAKE_INSTALL_PREFIX}/lib/orocos${OROCOS_PKG_DIR}/types")
```
Comment (goldhoorn) [inline lib/orogen/templates/typekit/CMakeLists.txt]: @smits these lines was added by you, are they still needed or was this a (old) nonworking workaround for cmake?

**Comment by goldhoorn** [comment]
no negative impact on linux, okay then for merge if @smits has no doubts

**Comment by smits** [comment]
I really want to test this, because rpath handling on OSX has already cause me many headaches. I'll test ASAP

**Comment by goldhoorn** [comment]
@smits: waiting for your okay ;)

**Comment by smits** [comment]
LGTM, I'm currently working on a branch that generates all orogen artefacts (except the templates to be filled in by the user) into the build dir which also does not have the old RPATH settings anymore.

Sorry for the long delay.

**Comment by goldhoorn** [comment]
On 11.08.2015 16:19, Ruben Smits wrote:
>
> LGTM, I'm currently working on a branch that generates all orogen 
> artefacts (except the templates to be filled in by the user) into the 
> build dir which also does not have the old RPATH settings anymore.
>
> Sorry for the long delay.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/orocos-toolchain/orogen/pull/55#issuecomment-129900077>.
>
@doudou ok for merge to you too?

-- 
  Dipl.-Inf. Matthias Goldhoorn
  Space and Underwater Robotic

  Universität Bremen
  FB 3 - Mathematik und Informatik
  AG Robotik
  Robert-Hooke-Straße 1
  28359 Bremen, Germany
  
  Zentrale: +49 421 178 45-6611
  
  Besuchsadresse der Nebengeschäftstelle:
  Robert-Hooke-Straße 5
  28359 Bremen, Germany
  
  Tel.:    +49 421 178 45-4193
  Empfang: +49 421 178 45-6600
  Fax:     +49 421 178 45-4150
  E-Mail:  matthias.goldhoorn@informatik.uni-bremen.de

  Weitere Informationen: http://www.informatik.uni-bremen.de/robotik

**Pull request merged**